content/learn/manuals.md

```diff
@@ -8,11 +8,11 @@ layout: "manuals"
 
 <ul id="links" class="list-unstyled version">
  <li>
-  <span class="mwl"><a href="/grass80/manuals/index.html " target="_blank"> GRASS GIS 8.0 (current) </a></span>
-  <a href="/grass80/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a> <a href="/grass80/manuals/grass-8.0_html_manual.zip" class="inl btn btn-secondary">Download ZIP</a>
+  <span class="mwl"><a href="/grass82/manuals/index.html " target="_blank"> GRASS GIS {{< grassVersion version="current" type="short">}}  (current) </a></span>
+  <a href="/grass82/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a> <a href="/grass82/manuals/grass-8.2_html_manual.zip" class="inl btn btn-secondary">Download ZIP</a>
```

Comment:
Can the URLs be driven by the variables, too? `8.2` is already there. `82` can be perhaps another variable. There are two there already, so this would be a third one. (Ultimately, it seems to me that the URLs is what needs to be fixed, i.e., there should be 8.2 as elsewhere.)

Reply:
> (Ultimately, it seems to me that the URLs is what needs to be fixed, i.e., there should be 8.2 as elsewhere.)

While I agree that URLs should be changed to be based on variables as well, I humbly disagree that they should include periods. Myriads of documents refer to GRASS GIS versions without dots, and breaking with the URL style after decades doesn't seem very useful to me.

Reply:
@wenzeslaus , I've added the 82 to the variables and used in the links and stuff. Kept them as they were.

Reply:
> > (Ultimately, it seems to me that the URLs is what needs to be fixed, i.e., there should be 8.2 as elsewhere.)
> 
> ...Myriads of documents refer to GRASS GIS versions without dots, and breaking with the URL style after decades doesn't seem very useful to me.

Out of scope for this PR, but would that be actually a problem? If they need to be updated, what is the difference between changing from `82` to `83` and changing from `82` to `8.3`? If they don't need to be updated, keeping the old URL is enough to keep the old thing working. If you would still need URLs from both the new and old system, there is likely no issue mixing both because you need to specify all numbers explicitly anyway like for the variables in this PR.

`10.0` seems more clear than `100`. `8.11` is definitively more clear than `811`. I think using dots everywhere is more predictable overall because it is consistent for all numbers within one version, while leaving out the dots leads to inconsistent numbers within one version (although being consistent across versions). However, clarity is anyway likely more important than consistency.

Reply:
> Out of scope for this PR, but would that be actually a problem?

Please, let's move this discussion to `grass-dev` as I would like to hear more voices here. In this repo it is too much hidden.

Reply:
> Out of scope for this PR, but would that be actually a problem? If they need to be updated, what is the difference between changing from `82` to `83` and changing from `82` to `8.3`?

I'd agree with Vaclav. Although it can be discussed on -dev ML,
1. somebody must make a post there;
2. there must be a short deadline to not delay this PR forever.